ambry-store/src/test/java/com.github.ambry.store/CompactionManagerTest.java

```diff
@@ -48,9 +48,8 @@
 
   /**
    * Instantiates {@link CompactionManagerTest} with the required cast
-   * @throws InterruptedException
```

Comment:
What's the code coverage for the new code in these tests?

Reply:
`testCompactionWithMisbehavingStores` method tests `addBlobStore()` method in CompactionManager. The intention is to ensure that compaction won't be scheduled on new added store until it is enabled.

Reply:
sorry I meant in percentage for all tests

Reply:
Like are all new code lines covered

Reply:
reviewed offline, new tests will be added to get to 100%